From 791284875324b192b1a670f16e8de0af2cde7ff2 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 31 Mar 2022 15:13:49 +0000
Subject: [PATCH] GCC: remove FMA initialization for
 segmentation_platform::UMAFeature

It is not allowed to initialize flexible member arrays. GCC has
an extension to allow it for static members. However, it is
not useful here and Clang ignores the initialization.
---
 .../default_model/query_tiles_model.cc                      | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/chrome/browser/segmentation_platform/default_model/query_tiles_model.cc b/chrome/browser/segmentation_platform/default_model/query_tiles_model.cc
index cd2d902..be86e87 100644
--- a/chrome/browser/segmentation_platform/default_model/query_tiles_model.cc
+++ b/chrome/browser/segmentation_platform/default_model/query_tiles_model.cc
@@ -48,15 +48,13 @@ constexpr UMAFeature kQueryTilesUMAFeatures[2] = {
                .bucket_count = 7,
                .tensor_length = 1,
                .aggregation = proto::Aggregation::COUNT,
-               .enum_ids_size = 0,
-               .accepted_enum_ids = {}},
+               .enum_ids_size = 0},
     UMAFeature{.signal_type = proto::SignalType::USER_ACTION,
                .name = "Search.QueryTiles.NTP.Tile.Clicked",
                .bucket_count = 7,
                .tensor_length = 1,
                .aggregation = proto::Aggregation::COUNT,
-               .enum_ids_size = 0,
-               .accepted_enum_ids = {}}};
+               .enum_ids_size = 0}};
 
 void AddUmaFeature(proto::SegmentationModelMetadata* metadata,
                    const UMAFeature features[],
-- 
2.34.1

